index.js

```diff
@@ -218,7 +218,7 @@ exports.writeMainView = function (dir, cb) {
 exports.writeIcon = function (dir, cb) {
   var iconPath = path.join(__dirname, 'assets/icon.png')
```

Comment:
@toddself this seems to work fine in a Windows 10 VM:

```
User@WinDev1704Eval MINGW64 ~/lab
$ echo "module.exports = () => console.log('hi')" > local.js

User@WinDev1704Eval MINGW64 ~/lab
$ node
> require('./local.js')()
hi
undefined
> require('../lab/local.js')()
hi
undefined
> require(require('path').join('../lab', './local.js'))()
hi
undefined
>
```

Tried it in the bash emulator git installs for you, as well as the default windows command prompt. So I guess our fear of `/` may be a holdover from early node days?